arches_component_lab/src/arches_component_lab/widgets/FileListWidget/components/FileListWidgetEditor/FileListWidgetEditor.vue

```diff
@@ -84,18 +85,27 @@ function onRemoveSavedFile(field: unknown, fileIndex: number): void {
         value: [...savedFiles.value, ...pendingFiles.value],
     });
 }
+
+function transformValueForForm(event: FormFieldResolverOptions) {
+    return {
+        display_value: event.value,
+        node_value: event.value,
+        details: [],
+    };
+}
 </script>
 
 <template>
     <GenericFormField
         v-bind="$attrs"
         :node-alias="nodeAlias"
-        :initial-value="value?.node_value"
+        :transform-value="transformValueForForm"
     >
         <template #default="$field">
             <FileUpload
                 :accept="allowedFileTypes"
                 :name="nodeAlias"
+                :model-value="value.node_value"
```

Comment:
handle for null "value"

```suggestion
                :model-value="value?.node_value"
```